action/protocol/staking/protocol.go

```diff
@@ -16,19 +16,17 @@ import (
 	"github.com/iotexproject/iotex-core/action"
 	"github.com/iotexproject/iotex-core/action/protocol"
 	"github.com/iotexproject/iotex-core/pkg/log"
+	"github.com/iotexproject/iotex-core/state/factory"
 )
 
-// protocolID is the protocol ID
```

Comment:
they are just happened to be the same value.

Reply:
yes, so just use the same value, instead of adding a new one

is this a concern?